magento/pwa-studio
# [Docs] update devdocs dependencies and build script

## Description

This PR updates Ruby dependencies in the lock file and fixes a broken production build script.

## Related Issue

None
## Acceptance 

@jcalcaben 

### Verification Stakeholders

@dpatil-magento 

### Specification



### Verification Steps

1. Navigate to the pwa-devdocs directory
1. Update ruby dependencies: `bundle install`
1. Verify command updated the ruby dependencies based on the lockfile changes
1. Create a production build: `npm run build:production`
1. Verify the build succeeds with no errors

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
